lib/main.dart

```diff
@@ -186,16 +240,27 @@ class _MyAppState extends State<MyApp> {
 
   @override
   void initState() {
+    WidgetsBinding.instance.addObserver(this);
+    AnalyticsService mixPanelService = GetIt.instance<AnalyticsService>();
+    mixPanelService.init();
     getHasFilled();
     loadFirstTime();
     super.initState();
+    _startSession();
+    //initMixpanel();
     personalPlanPhonesPageData = {
       'phoneName': <String>[],
       'emergencyPhones': <String>[],
       'phoneDescription': <String>[]
     }; // Initialize personalPlanPhonesPageData
   }
 
+  Future<void> initMixpanel() async {
+    // Once you've called this method once, you can access `mixpanel` throughout the rest of your application.
+    mixpanel = await Mixpanel.init("e38d39b73bc076129d0a5390af41fc24",
```

Comment:
Is this for testing?
seems it is not being called
maybe get the needed info from a (environment?) varible so you can use it locally and deploy it?